### DISCUSSION TITLE
How do I find proper info to add VLA S Band (3GHz) filter file to CIGALE?

### DISCUSSION THREAD

**Discussion by ralphy1017** [mboquien/cigale#58] [Q&A] [answered]
Hello all,

I am an undergraduate astrophysicist and I need some help and guidance finding the correct info for the radio transmission in the S band. I have never worked with Radio data before and I am unfamiliar with it. 

How can I add the correct transmission info to CIGALE? I briefly viewed the C band and L band that come with CIGALE and notice there are only 5 wavelength values and 5 transmission values. I don't know what to make of it.

Thanks in advance,
Ralph

**Comment by Guang91** [comment, marked as answer]
You can follow the format of the C/L band, i.e., assuming a boxcar shape (first convert your radio band frequency to wavelength, and then manually key in a filter file like the C/L band). It's an approximation anyway.

**Reply by ralphy1017** [reply]
Thanks!
